### DISCUSSION TITLE
plot scRNA cells co-embedded with the ATAC UMAP

### DISCUSSION THREAD

**Discussion by boconnell89** [GreenleafLab/ArchR#1697] [Questions / Documentation]
I'm wondering if there is a way either: plot scRNA cells that were used for gene integration, co-embedded with the UMAP, or else color the jointCCA UMAP's by another category (e.g. gene module score).

The motivation is to show that there is concordance between scRNA and gene module scores in cell type identification.  

Alternate ways of demonstrating the same are also good.

**Comment by rcorces** [comment]
Sorry for the delay in my reply.

> plot scRNA cells that were used for gene integration, co-embedded with the UMAP

If I'm understanding correctly, no this is not possible. 

> or else color the jointCCA UMAP's by another category (e.g. gene module score)

I think this previous issue post might be useful to you. https://github.com/GreenleafLab/ArchR/issues/373

> The motivation is to show that there is concordance between scRNA and gene module scores in cell type identification.

This seems like it could be done post-hoc. Find the cells from scRNA-seq that were used for integration and plot the correlation per-cell of the gene module score from ATAC) RNA. is that what you are hoping to accomplish? Apologies if I'm misinterpreting.
